ui/raidboss/data/02-arr/trial/titan-nm.txt

```diff
@@ -0,0 +1,105 @@
+### Titan Story / Normal
+# -ii 5C5 3CD -p 28B:203.0  28C:603.0
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+### Phase 1 100->85%
+0.0 "Start"
+0.0 "--sync--" sync /Engage!/ window 0,1
+2.0 "--sync--" sync /:Titan:368:/ window 2,0
+5.0 "Rock Buster" sync /:Titan:281:/

```

Comment:
Can you also add a `window 5,5` to this line? I usually like to sync the first non-auto as well, since it's not guaranteed that the boss always gets an auto off.

Reply:
Sure can